submit-pipeline/action.yml

```diff
@@ -20,7 +20,10 @@ inputs:
     description: The name of the pipeline version. Defaults to {PIPELINE_FUNCTION}_{GITHUB_SHA}.
     required: false
   PIPELINE_PARAMETERS_PATH:
-    description: Path to a parameters file in your repo; the parameters will be passed to the pipeline.
+    description: Path to a parameters YAML file in your repo; the parameters will be passed to the pipeline.
+    required: false
+  PIPELINE_PARAMETERS:
```

Comment:
To clarify my understanding this is a feature to support both pass in pipeline params a in-line yaml in addition to supporting passing in a yaml file as pipeline params

Reply:
Correct, example in progress here: https://github.com/Unity-Technologies/ml-platform-demos/blob/actions-custom-parameters/.github/workflows/submit-pipeline.yml